### DISCUSSION TITLE
Add Negative DIsplay option to moded 0.21

### DISCUSSION THREAD

**Discussion by LYCAON68** [egzumer/uv-k5-firmware-custom#322] [Q&A]
Add Negative display option to modded 0.21 : 
Is it possible to add a negative display option switch to  the modded 0.21 uv-k5 firmware or on the next release ?
Thank's ...

**Comment by egzumer** [comment]
I don't like it, but if someone else implements it I'll accept PR as an extra custom build option

**Reply by LYCAON68** [reply]
Thank's for Your feedback and further FW development, i just seen that previous versions implements this option, but it's only cosmetic not vital !
Best Regard's ...
